.github/workflows/ci-build.yml

```diff
@@ -42,7 +37,7 @@ jobs:
           compiler: gcc
           cxxcompiler: g++
           libraries: grfcodec
-          extra-cmake-parameters: -DOPTION_DEDICATED=ON -DCMAKE_CXX_FLAGS_INIT="-DRANDOM_DEBUG" -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
+          cmake-parameters: -DOPTION_DEDICATED=ON -DCMAKE_CXX_FLAGS_INIT="-DRANDOM_DEBUG" -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
```

Comment:
No, please keep this `extra`. In this context, that is actually correct :) I meant only the MacOS one ;)